app/src/lib/stores/app-store.ts

```diff
@@ -4551,7 +4560,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
     // Warn the user if there are changes in the working directory
```

Comment:
Could we add another comment here to explain why this condition? Something like…
```suggestion
    // Warn the user if there are changes in the working directory
    // This warning can be disabled, except when the user tries to undo
    // a merge commit.
```

Reply:
done